rasa/core/trackers.py

```diff
@@ -184,10 +184,23 @@ def current_state(
             "paused": self.is_paused(),
             "events": evts,
             "latest_input_channel": self.get_latest_input_channel(),
-            "active_form": self.active_form,
+            # TODO: Should we add a `active_loop` key and provide both keys for a while?
```

Comment:
@Ghostvv wdyt?

Reply:
I think so yes, maybe with deprecation warning. We need also to think carefully about `Form` event and how do we want to modify it with respect to the loop

Reply:
let's do it later when we actually decide whether to rename the event and so on